src/main/java/org/ibp/api/java/impl/middleware/dataset/AbstractDatasetExportService.java

```diff
@@ -182,6 +199,12 @@ File getReturnFile(final Study study, final List<File> files) throws IOException
 		return columns;
 	}
 
+	protected void addStockIdColumn(final List<MeasurementVariable> plotDataSetColumns) {
+		final Optional<MeasurementVariable>
+			designationColumn = plotDataSetColumns.stream().filter(measurementVariable -> measurementVariable.getName().equals("DESIGNATION")).findFirst();
```

Comment:
same here, should not be better to compare by id using TermId ? @cuenyad

Reply:
Done.